pymc4/tests/test_template_context.py

```diff
@@ -28,7 +28,7 @@ def test_free_forward_context_add_variable(monkeypatch):
     )
 
     with pytest.raises(Exception) as err:
-        random_variables.Normal("test_normal", loc=0, scale=1)
+        random_variables.Normal("test_normal", mu=0, sigma=1)
```

Comment:
I know I'm late to the merge by why sigma instead of sd like pymc3?

https://docs.pymc.io/api/distributions/continuous.html#pymc3.distributions.continuous.Normal

Reply:
They were three equivalent parameterizations. I think I chose to put in sigma because it matches the greek mathematical notation. I was used to both before (for whatever reason, sometimes sd was more natural, and at other times, sigma was more natural).

Reply:
Yes, 100% sigma. I made that mistake originally in pymc3 and it's just not consistent. Either mean and sd, or mu and sigma, or loc and scale, but never a mix.